### ISSUE TITLE
the Info.plist of iOS project from source code can't parse

### ISSUE THREAD

**Issue by magaofei** [MobSF/Mobile-Security-Framework-MobSF#699]
## STEPS TO REPRODUCE THE ISSUE

```
Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/plist_analysis.py:200
```
In my iOS project, the Info.plist file name is `Info.plist`

https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/AboutInformationPropertyListFiles.html
## CONSOLE OUTPUT

```
[2018-08-27 08:30:56]
[ERROR] Updating DB (/Users/mark/GitHub/downloads/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/db_interaction.py, LINE 248 "INFOPLIST=info_dict["plist_xml"],"): 'NoneType' object is not subscriptable

[2018-08-27 08:30:56]
[ERROR] Rendering to Template (/Users/mark/GitHub/downloads/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/db_interaction.py, LINE 150 "'plist': info_dict["plist_xml"],"): 'NoneType' object is not subscriptable
```

**Comment by superpoussin22** [comment]
@magaofei  duplicate from https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/686 ?

**Comment by magaofei** [comment]
@superpoussin22 yeah

**Comment by superpoussin22** [comment]
so I close #686 to have only one topic

**Comment by ajinabraham** [comment]
Closed with #700
